src/3p-frame-messaging.js

```diff
@@ -36,12 +36,17 @@ export const MessageType = {
   EMBED_SIZE_DENIED: 'embed-size-denied',
   NO_CONTENT: 'no-content',
 
+  // For the frame to be placed in full overlay mode for lightboxes
+  FULL_OVERLAY_FRAME: 'full-overlay-frame',
+  FULL_OVERLAY_FRAME_RESPONSE: 'full-overlay-frame-response',
+  RESET_FULL_OVERLAY_FRAME: 'reset-full-overlay-frame',
```

Comment:
I don't like "fullscreen" specifically because it's not really fullscreen. It fills the entire viewport, but the browser chrome may still be visible. Changed reset to cancel.